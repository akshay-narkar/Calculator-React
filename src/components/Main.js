export default function Main() {
  const heading = {
    color: 'black',
    fontSize: 50,
    marginBottom: 50,
  };

  const para = {
    fontSize: 20,
    marginBottom: 30,
  };

  return (
    <>
      <h1 style={heading}>Welcome to our page!</h1>

      <p style={para}>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit,
        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat.
        Duis aute irure dolor in reprehenderit in voluptate velit esse
        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
        cupidatat non proident, sunt in culpa qui o!cia deserunt mollit
        anim id est laborum. Ipsu
      </p>

      <p style={para}>
        Lorem ipsum dolor sit amet, consectetur adipisicing elit,
        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
        Ut enim ad minim veniam, quis nostrud exercitation ullamco
        laboris nisi ut aliquip ex ea commodo consequat.
        Duis aute irure dolor in reprehenderit in voluptate velit esse
        cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
        cupidatat non proident, sunt in culpa qui o!cia deserunt mollit
        anim id est laborum. Ipsu
      </p>
    </>

  );
}
