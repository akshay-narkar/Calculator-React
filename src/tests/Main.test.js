import renderer from 'react-test-renderer';
import Main from '../components/Main'

it('UI is unaltered wrt first snapshot', () => {

  const elem = renderer.create(<Main />).toJSON();

  expect(elem).toMatchSnapshot();

});