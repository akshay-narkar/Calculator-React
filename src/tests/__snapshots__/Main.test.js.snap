// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI is unaltered wrt first snapshot 1`] = `
Array [
  <h1
    style={
      Object {
        "color": "black",
        "fontSize": 50,
        "marginBottom": 50,
      }
    }
  >
    Welcome to our page!
  </h1>,
  <p
    style={
      Object {
        "fontSize": 20,
        "marginBottom": 30,
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>,
  <p
    style={
      Object {
        "fontSize": 20,
        "marginBottom": 30,
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>,
]
`;
